test/crd_validation_test.go

```diff
@@ -65,6 +65,7 @@ func validateCustomResources(t *testing.T, root string, crd string, prefix strin
 }
 
 func TestCompleteCRD(t *testing.T) {
+	t.Skip("Skipping CRD Validation")
```

Comment:
Is not better to write a small msg here why we are skipping, like commit-msg. and TODO also